AnnoyancesFilter/sections/css_extended.txt

```diff
@@ -93,6 +93,7 @@ facebook.com,facebookcorewwwi.onion#%#(function() { var observer=new MutationObs
 !! disabled due to rule update in English move rule to English filter if still nothing broken on 31.10.2018
 ! facebook.com##div[id^="substream_"] div[id^="hyperfeed_story_id_"][-ext-has='[id*="feed_subtitle_"] a[href^="https://www.facebook.com/#"] > span > span']
 ! End Facebook
+otpbank.hu#$##modal-content:contains('cookie') { display: none !important }
```

Comment:
Remove the mentioned rule and add this rule to cookies:
otpbank.hu#$##gdpr_modal { display:none!important; }